src/pages/RepoPage/CoverageOnboarding/CircleCI/CircleCI.tsx

```diff
@@ -38,110 +38,142 @@ function CircleCI() {
     enabled: showOrgToken,
   })
 
-  const uploadToken = orgUploadToken ?? data?.repository?.uploadToken
+  const uploadToken = orgUploadToken ?? data?.repository?.uploadToken ?? ''
```

Comment:
Should we show an error in the case that there is no valid upload token?

Reply:
Idk. I'm pretty sure that shouldn't happen. Can do just in case, but may not be worth effort. What do you think?

Reply:
Hrrm, okay chatting w/the platform team now, let's move forward with it as is and we can revisit later